cli/script/command-executor.ts

```diff
@@ -1020,6 +1055,50 @@ function getPackageMetricsString(packageObject: PackageWithMetrics): string {
     return returnString;
 }
 
+function getReactNativeProjectAppVersion(platform: string, projectName: string): Promise<string> {
+    if (platform === "ios") {
+        try {
+            var infoPlistContents: string = fs.readFileSync(path.join("iOS", projectName, "Info.plist")).toString();
+        } catch (err) {
+            try {
+                infoPlistContents = fs.readFileSync(path.join("iOS", "Info.plist")).toString();
+            } catch (err) {
+                throw new Error(`Unable to find or read "Info.plist" in the "iOS/${projectName}" or "iOS" folders.`);
+            }
+        }
+        
+        var appVersionRegex: RegExp = /CFBundleShortVersionString\s*<\s*\/key\s*>\s*<\s*string\s*>(.*?)<\s*\/string\s*>/;
+        var parsedInfoPlist: string[] = infoPlistContents.match(appVersionRegex);
```

Comment:
Isn't there an xml node module that we can use to parse the appVersion? Are you fairly confident that this RegEx won't break down the line if/when the format for info.plist changes?